cli/tests/extensions/permissions.rs

```diff
@@ -114,3 +116,20 @@ pub async fn get_package_details() {
         .failure()
         .stderr("❗ Error: Requires net access to \"phylum.io\"\n");
 }
+
+#[test]
+fn permission_op() {
+    let test_cli = TestCli::builder().with_config(None).build();
+
+    let permissions =
+        Permissions { read: Permission::List(vec!["/tmp".to_string()]), ..Permissions::default() };
+
+    let permissions_ext = "
+         const perms = PhylumApi.permissions()
+         console.log(perms);";
+
+    let status =
+        test_cli.extension(permissions_ext).with_permissions(permissions).build().run().success();
```

Comment:
Not sure what is going on here. Surely `rustfmt` should catch this?

Reply:
I'm not sure we have a proper lint for that -- I run `cargo +nightly fmt --all --check` as a pre commit hook, and even running it manually this still gets formatted this way.

Reply:
Nvm -- added stdout assertions and the line length tipped over and it's now nice and tidy.